safehttp/response_writer.go

```diff
@@ -89,26 +89,10 @@ func (w *ResponseWriter) NoContent() Result {
 	return Result{}
 }
 
-// ClientError TODO
-func (w *ResponseWriter) ClientError(code StatusCode) Result {
+// WriteError writes an error response (4xx, 5xx) according to the provided status
```

Comment:
Technically, the user could provide any status code so maybe make it more explicit that a valid status code should be between 400 and 599.